src/request_manager.ts

```diff
@@ -153,7 +159,18 @@ export class RequestManager {
     return result.data
   }
 
-  async tradeHistory(token: string, blockchain: string) : Promise<object> {
+  async getRSI(token : string, blockchain: string, periods : number | boolean = false) : Promise<number> {
+      if (typeof periods === 'boolean') periods = 14
+      let ohlcvdata : Ohlcv[] = await this.ohlcv(token, blockchain)
+      let period = ohlcvdata.slice(periods * -1)
+      let avgOpen = meanBy(period, (p) => Number(p.open))
+      let avgClose = meanBy(period, (p) => Number(p.close))
+      let RS = avgClose / avgOpen
+      let RSI = 100 - (100 / (1 + RS))
+      return RSI
+  }
```

Comment:
This function [does not belong](https://en.wikipedia.org/wiki/Unix_philosophy) in saturn.js 

The goal of saturn.js is to provide a wrapper around smart contracts and Saturn API, to make it a plugin into more complex systems like exchanges, wallets, trading bots.

Technical indicators belong in a independent technical indicator package